collect_app/src/main/java/org/odk/collect/android/utilities/StringUtils.java

```diff
@@ -0,0 +1,21 @@
+package org.odk.collect.android.utilities;
+
+import static java.lang.Character.isWhitespace;
+
+public class StringUtils {
```

Comment:
What would you think of using the existing `TextUtils` class? It has some other methods that will eventually come from `String`. (`isBlank` is in Java 11 but not yet in Android).

Reply:
Could we rename `TextUtils` to `StringUtils`? Personally would have made that much more discoverable for me.

Reply:
Works for me. It's kind of a random grab bag of things currently so I could also see an argument for moving `join` to your new `StringUtils` if you prefer.

Reply:
(Basically, fine to leave as-is, I just wanted to make sure you'd considered `TextUtils`)

Reply:
I'll rename to `StringUtils` and keep everything there.